specifyweb/frontend/js_src/lib/components/TreeView/Actions.tsx

```diff
@@ -157,6 +167,17 @@ export function TreeViewActions<SCHEMA extends AnyTree>({
           />
         </li>
       )}
+      {tableName === 'Storage' &&
+      hasPermission(resourceName as '/tree/edit/storage', 'bulk_move') ? (
+        <li className="contents">
+          <Button.Icon
+            disabled={disableButtons || isRoot}
```

Comment:
What is the reason to disable the bulk move action on the root node? 
At least from a functional standpoint, there is nothing wrong with allowing bulk moving preparations from the root node to other nodes. 

```suggestion
            disabled={disableButtons}
```